frontend/src/components/phones/onboarding/RelayNumberPicker.tsx

```diff
@@ -103,35 +95,26 @@ const RelayNumberSelection = (props: RelayNumberSelectionProps) => {
   const [relayNumberIndex, setRelayNumberIndex] = useState(0);
 
   useEffect(() => {
-    _getRelayNumberSuggestions();
-  }, []);
-
-  // GET request to get relay number suggestions
-  const _getRelayNumberSuggestions = () => {
-    setIsLoading(true);
-    getRelayNumberSuggestions()
-      .then((response) => response.json())
-      .then((data) => {
-        const { other_areas_options, same_area_options, same_prefix_options } =
-          data;
-        const suggestedNumbers: RelayNumberOption[] = [
-          ...same_area_options,
-          ...same_prefix_options,
-          ...other_areas_options,
-        ];
+    if (relayNumberSuggestionsData.data) {
+      // get all relay number suggestions from our API
+      const { other_areas_options, same_area_options, same_prefix_options } =
+        relayNumberSuggestionsData.data;
+      // combine all relay number suggestions into one array
+      const suggestedNumbers = [
+        ...same_area_options,
+        ...other_areas_options,
+        ...same_prefix_options,
+      ];
 
-        setRelayNumberSuggestions(
-          suggestedNumbers.map(
-            (suggestion: RelayNumberOption) => suggestion.phone_number
-          )
-        );
+      // set the relay number suggestions - an array of strings
+      setRelayNumberSuggestions(
+        suggestedNumbers.map((suggestion) => suggestion.phone_number)
+      );
```

Comment:
You also don't need a separate state variable for this if it's outside of the effect; you can just directly do:

```ts
const relayNumberSuggestions = `relayNumberSuggestionsData.data?.map(/* ... */) ?? [];
```

Reply:
Done - I went ahead and implemented something very similar.